scripts/upload-apk.sh

```diff
@@ -14,6 +14,9 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/neuro
     exit 0
 fi
 
+# Generate Playstore bundle
+./gradlew bundlePlaystoreRelease
```

Comment:
Copy pasted from PSLab right away I guess. This is invalid. Change to `./gradlew bundleRelease`